Anytype/Sources/PresentationLayer/TextEditor/EditorPage/EditorPageViewModel+ChangeType.swift

```diff
@@ -1,13 +1,27 @@
 import Services
+import AnytypeCore
+
 
 extension EditorPageViewModel {
```

Comment:
Extension in different place will be by problems when we want to use private fields